### ISSUE TITLE
Don't call the api if a sensor is already installed

### ISSUE THREAD

**Issue by ffalor** [CrowdStrike/aws-ssm-distributor#29]
Add a install check to windows/linux to exit early if the sensor is already installed

example:

```powershell
$agentService = Get-Service -Name CSAgent -ErrorAction SilentlyContinue

if ($agentService) {
    Write-Output 'Falcon Sensor already installed... if you want to update or downgrade, please use Sensor Update Policies in the CrowdStrike console.'
    Exit 0
}
```

**Comment by ffalor** [comment]
this also fixes #24

**Comment by ffalor** [comment]
This has been merged - will be released on the next release
